source/features/redirect-to-issue-list.tsx

```diff
@@ -0,0 +1,41 @@
+import * as pageDetect from 'github-url-detection';
+import delegate, {DelegateEvent} from 'delegate-it';
+
+import features from '../feature-manager.js';
+
+function redirectToIssues(event: DelegateEvent<Event, HTMLFormElement>): void {
+	event.preventDefault();
+
+	const form = event.delegateTarget;
+	const searchInput = form.elements['js-issues-search'];
+
+	if (!searchInput.value.includes('is:pr')) {
```

Comment:
Should we make this check more precise ? e.g. `value.split(/ +/g).includes('is:pr')`

Reply:
We actually have `SearchQuery` to deal with this